### PR TITLE
post How rOpenSci runs community calls

### PR DESCRIPTION
**Pull request by stefaniebutland**

### CODE REVIEW

File: content/blog/2021-02-02-ropensci-community-calls/index.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+tags:
+  - community call
+  - events
+description: "Tips from choosing a topic to capturing outcomes"
+preface: "In December 2020, Stefanie Butland, rOpenSci’s Community Manager, gave a presentation and Q & A session on community calls for Code for Science & Society Event Fund grantees and fiscally-sponsored project leaders. Emily Lescak, CS&S Event Fund Manager, led the discussion. This post is adapted from the original at GET LINK https://eventfund.codeforscience.org/blog/URL."
+twitterImg: blog/2021/02/02/ropensci-community-calls/commcalls-img.png
+twitterAlt: "collage of images from community calls with 3 Zoom views of speakers, a notes doc and view of someone's screen"
+---
+
+rOpenSci community calls are online events in which information is shared with the goals of reinforcing our mission and values, informing people on technical and social topics, bringing different stakeholders’s perspectives to light, and identifying unmet needs on a topic.
```
Comment (maelle) [inline content/blog/2021-02-02-ropensci-community-calls/index.md:17]:
```suggestion
rOpenSci community calls are online events in which information is shared with the goals of reinforcing our mission and values, informing people on technical and social topics, bringing different stakeholders' perspectives to light, and identifying unmet needs on a topic. 
```

I think the second s is not needed?

File: content/blog/2021-02-02-ropensci-community-calls/index.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+description: "Tips from choosing a topic to capturing outcomes"
+preface: "In December 2020, Stefanie Butland, rOpenSci’s Community Manager, gave a presentation and Q & A session on community calls for Code for Science & Society Event Fund grantees and fiscally-sponsored project leaders. Emily Lescak, CS&S Event Fund Manager, led the discussion. This post is adapted from the original at GET LINK https://eventfund.codeforscience.org/blog/URL."
+twitterImg: blog/2021/02/02/ropensci-community-calls/commcalls-img.png
+twitterAlt: "collage of images from community calls with 3 Zoom views of speakers, a notes doc and view of someone's screen"
+---
+
+rOpenSci community calls are online events in which information is shared with the goals of reinforcing our mission and values, informing people on technical and social topics, bringing different stakeholders’s perspectives to light, and identifying unmet needs on a topic. 
+Our calls can include one to six speakers, or a moderated panel discussion. 
+They run for an hour and consist of 40 minutes of presentation and 20 minutes for questions from the audience. 
+**Strong social facilitation at every step is our key to success**.
```
Comment (maelle) [inline content/blog/2021-02-02-ropensci-community-calls/index.md:20]: I'd put this sentence later.
Maybe add a question "Do you wonder what our recipe for success is?" after giving some measure of success (number of attendees, various backgrounds, folks inviting you to explain the recipe).

File: content/blog/2021-02-02-ropensci-community-calls/index.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+---
+
+rOpenSci community calls are online events in which information is shared with the goals of reinforcing our mission and values, informing people on technical and social topics, bringing different stakeholders’s perspectives to light, and identifying unmet needs on a topic. 
+Our calls can include one to six speakers, or a moderated panel discussion. 
+They run for an hour and consist of 40 minutes of presentation and 20 minutes for questions from the audience. 
+**Strong social facilitation at every step is our key to success**. 
+
+In this post, we’ll review the flow of community call development, from deciding on a topic to capturing the outcomes. 
+Many of the same principles apply, whether the event is called a community call, panel discussion, or similar. 
+Our goal is to share how we do it so you might adapt some elements to meet your needs.
```
Comment (maelle) [inline content/blog/2021-02-02-ropensci-community-calls/index.md:24]: could you rephrase "meet your needs" as there is "unmet needs" a bit earlier? sounds repetitive.

File: content/blog/2021-02-02-ropensci-community-calls/index.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+{{< figure src = "commcalls-img.png" class = "center" width = "350" alt = "collage of images from community calls with 3 Zoom views of speakers, a notes doc and view of someone's screen" >}}
+
+## Choosing a topic
+
+Typically, I (Stefanie) solicit ideas from the rOpenSci team and community. 
+It’s best to keep the topic focused rather than trying to cover too much. 
+Recently I've experimented with crowdsourcing ideas in a transparent way via [GitHub](https://github.com/ropensci-org/community-calls/issues). 
+This has resulted in some rich public discussions that help focus and prioritize topics. 
+You’re welcome to adapt this approach for your own needs; please acknowledge rOpenSci if you do.
```
Comment (maelle) [inline content/blog/2021-02-02-ropensci-community-calls/index.md:34]: should this be stated more generally instead of at this particular point?

File: content/blog/2021-02-02-ropensci-community-calls/index.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+{{< figure src = "commcalls-topics-gh-shadow.png" class = "center" width = "500" caption = "Excerpt of [suggested community call topics on GitHub](https://github.com/ropensci-org/community-calls/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc), labelled according to their status: 0/idea; 1/approved; 2/working on it; 3/completed" alt = "Excerpt of [suggested community call topics on GitHub](https://github.com/ropensci-org/community-calls/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc), labelled according to their status: 0/idea; 1/approved; 2/working on it; 3/completed" >}}
+
+
+To prioritize topics for the 2021 season, I pitched them to my colleagues and asked them to vote for their top five, using emojis on Slack.
```
Comment (maelle) [inline content/blog/2021-02-02-ropensci-community-calls/index.md:39]: I like emoji polls so much that it'd be worth a footnote with a fake example (not sure we can add pictures in footnotes though... maybe a tweet and you'd link to it. such a fun way to make folks vote)

File: content/blog/2021-02-02-ropensci-community-calls/index.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+The open topic discussions on GitHub can reveal potential speakers or people whom I might ask to suggest others who are not “the usual suspects”. 
+
+I start planning calls about two months in advance so that I can find a time that works for the presenters.
+I use a [scheduling tool](https://lettucemeet.com) to narrow down a date and time. 
+When scheduling, keep time zones in mind! Think about where your target audience is largely located and whether the time is welcoming for them to attend. 
+
+I schedule a one-hour pre-event meeting with speakers to introduce them to each other, have them share what they’re considering talking about, help them understand their audience, ask them what questions they think should be raised, and cover logistical details of the event. 
+While the panelists are experts in their respective areas, they are not necessarily experienced in communicating with this particular community. 
+I've found that these pre-event sessions lead to meaningful conversations that allow panelists to maximize their complementarity and give a sense of a comfortable discussion during the event. 
+I think this may actually end up minimizing their preparation time.
```
Comment (maelle) [inline content/blog/2021-02-02-ropensci-community-calls/index.md:54]: at least it surely help them make it more targeted?

File: content/blog/2021-02-02-ropensci-community-calls/index.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+I schedule a one-hour pre-event meeting with speakers to introduce them to each other, have them share what they’re considering talking about, help them understand their audience, ask them what questions they think should be raised, and cover logistical details of the event. 
+While the panelists are experts in their respective areas, they are not necessarily experienced in communicating with this particular community. 
+I've found that these pre-event sessions lead to meaningful conversations that allow panelists to maximize their complementarity and give a sense of a comfortable discussion during the event. 
+I think this may actually end up minimizing their preparation time. 
+
+## Promoting the event 
+
+Our  community calls are advertised on our [website](https://ropensci.org/commcalls/), Twitter, and relevant Slack groups and mailing lists. 
+With many people finding increased constraints on their time and more virtual events to choose from, I emphasize the specific value of each community call and what makes their content and style unique.
+Successful promotion benefits immensely from overlap between members of different communities and their interests.
```
Comment (maelle) [inline content/blog/2021-02-02-ropensci-community-calls/index.md:60]: I do not understand this sentence.

File: content/blog/2021-02-02-ropensci-community-calls/index.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+{{< figure src = "commcalls-promotion-shadow.png" class = "center" width = "600" caption = "Example promotional elements include speaker bios on the [landing page for the Community Call](https://ropensci.org/commcalls/dec2020-datarepos/) and a  [tweet](https://twitter.com/rOpenSci/status/1329092004496748545) that others can share" alt = "On left, a screenshot of 5 speaker bios with headshot and text for each. On right, screenshot of a [tweet](https://twitter.com/rOpenSci/status/1329092004496748545) tagging those speakers in promotion of the community call" >}}
+
+## Running the event
+
+It is important to begin with a script consisting of introducing your organization, code of conduct, Zoom etiquette, noting the call is recorded, norms for asking questions, and introductions to the speakers. Attendees are encouraged to add their questions and take notes in a [collaborative doc](http://bit.ly/ropensci-commcall-maintaining). 
+This results in clear and concise questions and can help lower barriers to participation.
```
Comment (maelle) [inline content/blog/2021-02-02-ropensci-community-calls/index.md:67]: Also isn't it easier to moderate when you can pick a question rather than having to let someone speak without knowing what they're going to ask?

File: content/blog/2021-02-02-ropensci-community-calls/index.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+
+## Running the event
+
+It is important to begin with a script consisting of introducing your organization, code of conduct, Zoom etiquette, noting the call is recorded, norms for asking questions, and introductions to the speakers. Attendees are encouraged to add their questions and take notes in a [collaborative doc](http://bit.ly/ropensci-commcall-maintaining). 
+This results in clear and concise questions and can help lower barriers to participation. 
+During Q&A, people are invited to unmute to ask their question, or the moderator can ask on their behalf. 
+Over time, this practice of typing questions in the shared doc has led to attendees adding their own responses to questions, yielding a [rich resource](/blog/2020/07/14/commcall-maintaining-pkg/) that harnesses distributed community knowledge.
+
+We typically get 40-50 attendees, sometimes 90, and recently had over 150. 
+It's important to think beyond numbers of attendees; even if there is a lower turnout, those in attendance are likely gaining a lot from the discussion.
```
Comment (maelle) [inline content/blog/2021-02-02-ropensci-community-calls/index.md:72]: should the value of recordings & other artefacts also be mentioned?

Comment (maelle) [reply content/blog/2021-02-02-ropensci-community-calls/index.md:72]: especially as it's a good transition to the next section.

File: content/blog/2021-02-02-ropensci-community-calls/index.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+
+*   [Landing page](/commcalls/dec2020-datarepos/) with all the things, including crowdsourced resources
+*   [Public discussion](https://github.com/ropensci-org/community-calls/issues/9) that informed developing the topic and choosing speakers;
+*   [Videos](https://vimeo.com/ropensci) of all past calls;
+*   [Collaborative notes](http://bit.ly/ropensci-commcall-datarepos);
+*   [Summary blog post](/blog/2020/07/14/commcall-maintaining-pkg/) example.
+
+{{< figure src = "commcalls-video-shadow.png" class = "center" width = "400" caption = "Screenshot from [video](/commcalls/dec2020-datarepos/) with closed captions" alt = "screenshot from a community call video shows speaker with slide behind him, and closed caption text in bottom center of image" >}}
+
+Last year, Janani Ravi a first-time attendee, went from adding her notes in our shared doc to having a byline on the rOpenSci blog.
```
Comment (maelle) [inline content/blog/2021-02-02-ropensci-community-calls/index.md:89]:
```suggestion
Last year, [Janani Ravi](/author/janani-ravi/), a first-time attendee, went from adding her notes in our shared doc to having a byline on the rOpenSci blog.
```

File: content/blog/2021-02-02-ropensci-community-calls/index.md [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+In her [summary post](/blog/2020/07/14/commcall-maintaining-pkg/) about Maintaining an R Package she said:
+
+> ...learning the right terminology/lingo/vocabulary is everything. It can take a few dozen blog posts and many hours of reading before beginners get to the ‘aha’ moment of ‘oh, these are the terms I need to use to search for what I’m thinking!’...There’s nothing like hearing a lively technical banter of experts to pick up the vernacular that one can easily build upon. That is the precise gap filled in by the timely and helpful rOpenSci Package Maintenance Community Call!
+
+Don’t underestimate the value of sharing your success stories or anecdotes with your team and your community!
```
Comment (maelle) [inline content/blog/2021-02-02-ropensci-community-calls/index.md:94]: this sentence has two meanings at different meta levels :-)

File: content/blog/2021-02-02-ropensci-community-calls/index.md [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+
+## Tips
```
Comment (maelle) [inline content/blog/2021-02-02-ropensci-community-calls/index.md:96]: is this the conclusion? should it be " a few last tips"?

File: content/blog/2021-02-02-ropensci-community-calls/index.md [hunk 13]
```diff
@@ -0,0 +1,104 @@
...
+
+Community calls are a great low-barrier way for new people to be exposed to your project and for you to hear what people are thinking about a topic. 
+It takes a lot of time to do them well! 
+It's ok to start simple and evolve. 
+To streamline the effort of organizing, work from a checklist. Where possible, be open and expose your ideas and progress to your community and encourage their input in the planning phase.
```
Comment (maelle) [inline content/blog/2021-02-02-ropensci-community-calls/index.md:101]: why not share some checklist starter?

File: content/blog/2021-02-02-ropensci-community-calls/index.md [hunk 14]
```diff
@@ -0,0 +1,104 @@
...
+Don’t underestimate the value of sharing your success stories or anecdotes with your team and your community!
+
+## Tips
+
+Community calls are a great low-barrier way for new people to be exposed to your project and for you to hear what people are thinking about a topic. 
+It takes a lot of time to do them well! 
+It's ok to start simple and evolve. 
+To streamline the effort of organizing, work from a checklist. Where possible, be open and expose your ideas and progress to your community and encourage their input in the planning phase. 
+Most importantly, be intentional. 
+Think about what it's like to be the speaker, or the person attending their first community call with your project and build a welcoming and valuable experience for them.
```
Comment (maelle) [inline content/blog/2021-02-02-ropensci-community-calls/index.md:103]: :blue_heart:

**Review by maelle — COMMENTED** [review]
Awesome post, very cool to see you share your thinking!

File: content/blog/2021-02-02-ropensci-community-calls/index.md (hunk 3 above)
Comment (maelle) [reply content/blog/2021-02-02-ropensci-community-calls/index.md:24]: "for your own endeavours" or so

File: content/blog/2021-02-02-ropensci-community-calls/index.md (hunk 4 above)
Comment (maelle) [reply content/blog/2021-02-02-ropensci-community-calls/index.md:34]: maybe also repeated in the conclusion, "we'd love to hear from you if you apply some of the tips"

File: content/blog/2021-02-02-ropensci-community-calls/index.md (hunk 7 above)
Comment (maelle) [reply content/blog/2021-02-02-ropensci-community-calls/index.md:60]: i.e. do you mean members will share in their own communities?

File: content/blog/2021-02-02-ropensci-community-calls/index.md [hunk 15]
```diff
@@ -0,0 +1,104 @@
...
+{{< figure src = "commcalls-topics-gh-shadow.png" class = "center" width = "500" caption = "Excerpt of [suggested community call topics on GitHub](https://github.com/ropensci-org/community-calls/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc), labelled according to their status: 0/idea; 1/approved; 2/working on it; 3/completed" alt = "Excerpt of [suggested community call topics on GitHub](https://github.com/ropensci-org/community-calls/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc), labelled according to their status: 0/idea; 1/approved; 2/working on it; 3/completed" >}}
+
+
+To prioritize topics for the 2021 season, I pitched them to my colleagues and asked them to vote for their top five, using emojis on Slack.
+
+## Selecting and preparing speakers
+
+Depending on the topic, it may make sense to invite a single speaker. 
+More often, several short presentations or a panel discussion is more appropriate, bearing in mind that organizing regular events with multiple speakers is time consuming! 
+The open topic discussions on GitHub can reveal potential speakers or people whom I might ask to suggest others who are not “the usual suspects”.
```
Comment (maelle) [inline content/blog/2021-02-02-ropensci-community-calls/index.md:45]: btw should this section mention something about selecting diverse speakers not only as regards the topic they cover?

File: content/blog/2021-02-02-ropensci-community-calls/index.md (hunk 3 above)
Comment (steffilazerte) [reply content/blog/2021-02-02-ropensci-community-calls/index.md:24]: I agree that a bit of re-wording would be useful here. It's not clear to me why the majority of rOpenSci users would need to know how to host a commcall... so I would suggest elaborating/re-phrasing  "meet your needs".

File: content/blog/2021-02-02-ropensci-community-calls/index.md (hunk 9 above)
Comment (steffilazerte) [reply content/blog/2021-02-02-ropensci-community-calls/index.md:72]: Consider a connecting word like "However, it's important to think beyond ..."

File: content/blog/2021-02-02-ropensci-community-calls/index.md [hunk 16]
```diff
@@ -0,0 +1,104 @@
...
+This results in clear and concise questions and can help lower barriers to participation. 
+During Q&A, people are invited to unmute to ask their question, or the moderator can ask on their behalf. 
+Over time, this practice of typing questions in the shared doc has led to attendees adding their own responses to questions, yielding a [rich resource](/blog/2020/07/14/commcall-maintaining-pkg/) that harnesses distributed community knowledge.
+
+We typically get 40-50 attendees, sometimes 90, and recently had over 150. 
+It's important to think beyond numbers of attendees; even if there is a lower turnout, those in attendance are likely gaining a lot from the discussion. 
+For example, our community call on [involving multilingual communities](/commcalls/2019-06-28/) had low attendance, but led directly to our experiment with doing [software peer review in Spanish](https://github.com/ropensci/software-review/issues/414).
```
Comment (steffilazerte) [inline content/blog/2021-02-02-ropensci-community-calls/index.md:73]:
```suggestion
For example, our community call on [involving multilingual communities](/commcalls/2019-06-28/) had relatively low attendance, but led directly to our experiment with doing [software peer review in Spanish](https://github.com/ropensci/software-review/issues/414).
```

File: content/blog/2021-02-02-ropensci-community-calls/index.md [hunk 17]
```diff
@@ -0,0 +1,104 @@
...
+
+## Capturing outcomes and impact
+
+The [landing page](/commcalls/dec2020-datarepos/) for each community call acts as an archive that includes the video with closed captions (ask me anything), links to speakers’ slides, collaborative notes doc, and resources on the topic.
```
Comment (steffilazerte) [inline content/blog/2021-02-02-ropensci-community-calls/index.md:77]: Ask me anything?

File: content/blog/2021-02-02-ropensci-community-calls/index.md (hunk 10 above)
Comment (steffilazerte) [reply content/blog/2021-02-02-ropensci-community-calls/index.md:89]: I like the comment about Janani, but I'm not sure it follows clearly from the previous paragraph. Perhaps start this section with a sentence or two of overview, linking the title of the section "Capturing outcomes and impact" directly to the content ("Artifacts" and personal stories)

File: content/blog/2021-02-02-ropensci-community-calls/index.md (hunk 4 above)
Comment (steffilazerte) [inline content/blog/2021-02-02-ropensci-community-calls/index.md:34]: Perhaps something a bit softer, considering we're an open source community. Maybe something along these lines:
```suggestion
If you find this useful and adapt this approach for your own needs we'd love an rOpenSci shoutout!
```

**Comment by steffilazerte** [comment]
Looks good! I've added my comments to @maelle's. Where I felt neutral I didn't comment, where I aggreed, I added my voice :) I also added my own comments to different sections. Structurally the only paragraph that I struggled with was the Capturing outcomes and impact (see my comments). 

Did you want a formal checklist review?

**Comment by stefaniebutland** [comment]
Thanks for the reviews @maelle and @steffilazerte! I won't need a checklist review. I'll do the author/reviewer stuff after I incorporate your feedback.

File: content/blog/2021-02-02-ropensci-community-calls/index.md (hunk 17 above)
Comment (stefaniebutland) [reply content/blog/2021-02-02-ropensci-community-calls/index.md]: adding footnote to explain, and omitting this txt

File: content/blog/2021-02-02-ropensci-community-calls/index.md (hunk 12 above)
Comment (stefaniebutland) [reply content/blog/2021-02-02-ropensci-community-calls/index.md]: revising to "Final thoughts"

File: content/blog/2021-02-02-ropensci-community-calls/index.md (hunk 6 above)
Comment (stefaniebutland) [reply content/blog/2021-02-02-ropensci-community-calls/index.md:54]: added "focussed" to "give a sense of a comfortable focussed discussion during the event"

File: content/blog/2021-02-02-ropensci-community-calls/index.md (hunk 7 above)
Comment (stefaniebutland) [reply content/blog/2021-02-02-ropensci-community-calls/index.md:60]: changed to "Successful promotion benefits immensely from people sharing with related communities to which they belong."

File: content/blog/2021-02-02-ropensci-community-calls/index.md (hunk 13 above)
Comment (stefaniebutland) [reply content/blog/2021-02-02-ropensci-community-calls/index.md]: Thanks @maelle for the suggestion. I added a [generalized checklist to our public repo](https://github.com/ropensci-org/community-calls/blob/master/checklist.md) and linked to it in this paragraph

File: content/blog/2021-02-02-ropensci-community-calls/index.md [hunk 18]
```diff
@@ -0,0 +1,111 @@
...
+tags:
+  - community call
+  - events
+description: "Tips from choosing a topic to capturing outcomes"
+preface: "In December 2020, Stefanie Butland, rOpenSci’s Community Manager, gave a presentation on community calls for Code for Science & Society Event Fund grantees and fiscally-sponsored project leaders. Emily Lescak, CS&S Event Fund Manager, led the discussion. This post builds on [the summary post](https://eventfund.codeforscience.org/community-call/) from that session."
+twitterImg: blog/2021/02/02/ropensci-community-calls/commcalls-img.png
+twitterAlt: "collage of images from community calls with 3 Zoom views of speakers, a notes doc and view of someone's screen"
+---
+
+rOpenSci community calls are online events in which information is shared with the goals of reinforcing our mission and values, informing people on technical and social topics, bringing different stakeholders’ perspectives to light, and identifying unmet needs on a topic.
```
Comment (steffilazerte) [inline content/blog/2021-02-02-ropensci-community-calls/index.md:17]:
```suggestion
rOpenSci community calls are online events with the fundamental goal of strengthening our community. We have the additional goals of reinforcing our mission and values, informing people on technical and social topics, bringing different stakeholders’ perspectives to light, and identifying unmet needs on a topic. 
```

File: content/blog/2021-02-02-ropensci-community-calls/index.md [hunk 19]
```diff
@@ -0,0 +1,111 @@
...
+
+rOpenSci community calls are online events in which information is shared with the goals of reinforcing our mission and values, informing people on technical and social topics, bringing different stakeholders’ perspectives to light, and identifying unmet needs on a topic. 
+Calls can include one to six speakers, or a moderated panel discussion. 
+They run for an hour and consist of 40 minutes of presentation and 20 minutes for questions from the audience. 
+They have attracted fairly large audiences[^1] with significant representation from academia, government, and industry. 
+Increasingly we have been asked to share our approach[^2]. 
+
+In this post, we review the flow of community call development, from deciding on a topic to capturing the outcomes. 
+Many of the same principles apply to a wide range of virtual events for community engagement. 
+Strong social facilitation at every step is a key to our success.
```
Comment (steffilazerte) [inline content/blog/2021-02-02-ropensci-community-calls/index.md:25]:
```suggestion
Generally, our calls attract fairly large audiences[^1] and we see significant representation from academia, government, and industry. 

To our delight, we have increasingly been asked to share our approach[^2].
Therefore, in this post, we review the flow of community call development, from deciding on a topic to capturing the outcomes. We also highlight the strong social facilitation at every step, which is a key to our success.
Many of the same principles apply to a wide range of virtual events for community engagement, so we hope these techniques will be broadly useful.

```

**Review by steffilazerte — COMMENTED** [review]
I found the intro a bit sterile and choppy. I have introduced looser, more personal language to soften it a bit, and have rearranged some sentences to improve flow.

Take what you like, leave what you don't :grin:

**Comment by stefaniebutland** [comment]
* [ ] I have read the Content Guidelines.
* [ ] I have read the Technical Guidelines.
* [ ] I used or followed the R Markdown or Markdown template.
* [ ] I have followed the Style Guide.
* [ ] I created or updated my author metadata with correct folder name.
* [ ] I have added relevant tags after browsing existing tags (incuding "community" tag).
* [ ] I ran `roblog::ro_lint_md()` on index.md (optional).
* [ ] I ran `roblog::ro_check_urls()` on index.md (optional).
* [ ] I ran a spell-check on index.md.
